ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: optional-operators-cvp-scorecard-test
+  from_image:
+    namespace: ci
+    name: cvp-operator-scorecard
+    tag: v1
+  commands: optional-operators-cvp-scorecard-test-commands.sh
+  resources:
+    requests:
+      cpu: 300m
+      memory: 300Mi
+  credentials:
+  - namespace: brew-registry-pullsecret # this entry injects the custom credential
```

Comment:
IIRC we [changed this](https://github.com/openshift/release/pull/16489) in the other step to use the standard `test-credentials` namespace where the secret is automatically synced across clusters (more standard config -> fewer suprises )

Reply:
Thanks for noticing, gonna change that.